src/containers/AdminExternalSystems.tsx

```diff
@@ -124,6 +128,17 @@ class AdminExternalSystems extends Component<Props, State> {
       externalSystem: { ...prevState.externalSystem, ...{ [prop]: newVal } }
     }));
 
+  handleCellClick = (row: number, column: number) => {
+    // Ignore clicks in Actions column
+    if (column > 2) return;
```

Comment:
Can we do something by column name here? Just expecting this to break if we reorder columns, etc.